index.html

```diff
@@ -76,14 +76,16 @@ <h2 id="title_main"></h2>
       <div id="isotope-container"></div>
       <div id="modals"></div>
     </div>
+
 		<script type="text/javascript" src="https://code.jquery.com/jquery-1.10.1.min.js"></script>
-    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.6.0/underscore-min.js"></script>
+        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.6.0/underscore-min.js"></script>
+		<script type="text/javascript" src="js/randomColor.js"></script>
```

Comment:
Please add the steps to regenerate this error as well

@lakindu95 please continue with this..